functions/Enums.js

```diff
@@ -164,3 +164,30 @@ exports.NotificationContentType = {
     DELIVERY_REQUEST: 'delivery_request',
     ACCOUNT: 'account'
 };
+
+exports.EmailTemplate = {
+    EMAIL_ALL: {
+        subject: 'Meal Matchup Delivery Confirmation',
+        html:'<html> <head> <title></title> </head> <body><p>Hello,</p><div> </div><span>The delivery that you are a part of has been confirmed by all 3 parties. To see the updated status of the delivery, please login here: </span><a href="https://www.mealmatchup.org/">https://www.mealmatchup.org/</a><span>.</span><p>Thank you!</p><p>Meal Matchup</p></body></html>'
```

Comment:
We might wanna change the url to: https://uw-foodwaste.web.app/ for all these for now since that's the live production app rather than mealmatchup.org